apps/ledger-live-desktop/src/renderer/screens/platform/LiveApp.tsx

```diff
@@ -55,7 +59,17 @@ export function LiveApp({ match, appId: propsAppId, location }: Props) {
     );
   }, [search, customDappUrl, urlParams?.customDappUrl, internalParams?.customDappUrl]);
 
-  const handleClose = useCallback(() => history.push(returnTo || `/platform`), [history, returnTo]);
+  const handleClose = useCallback(() => {
+    if (returnTo.indexOf("/swap") > -1) {
```

Comment:
```suggestion
    if (returnTo.startsWith("/swap")) {
```
This seems a bit more correct like that, unless I missed something

Reply:
Yeap, fair point, updated 👍